test/apps/iodemo/io_demo.cc

```diff
@@ -2525,7 +2573,7 @@ class DemoClient : public P2pDemoCommon {
     }
 
 private:
-    std::vector<server_info_t>              _server_info;
+    std::vector<server_info_t *>            _server_info;
```

Comment:
why converted to pointer?

Reply:
because coverity check reported "Big parameter passed by value" due to sizeof(server_info_t) increased.

Reply:
where is it passed as a parameter?

Reply:
does that mean we need to change `static double get_time();` to `static float get_time();`?

Reply:
> where is it passed as a parameter?

it seems vector element size limit. copy and paste here:

/usr/include/c++/4.8.5/bits/stl_vector.h:704:
  pass_by_value: Passing parameter __x of type "std::vector<DemoClient::server_info_t, std::allocator<DemoClient::server_info_t> >::value_type" (size 136 bytes) by value.

Reply:
no; change only max_lat/avg_lat

Reply:
done. using "float" instead avoids the coverity issue.

> @huaxiangfan i see that we don't have space before "MBs" . so can you please also remove the space before "us"? 10x

done. now it looks like
[1652359269.408880] [UCX-connection 0x204a0e0: #1 1.1.1.3:1337] read 371.476MBs iops: 95097 max-lat: 26.9413us avg-lat: 4.94058us | write 371.476MBs iops: 95097 max-lat: 21.9345us avg-lat: 5.04014us | total 742.952MBs iops: 190195 max-lat: 26.9413us avg-lat: 4.99036us